app/config/collections.php

```diff
@@ -769,46 +758,57 @@
         ],
     ],
 
-    'schedules' => [
+    'rules' => [
         '$collection' => ID::custom(Database::METADATA),
-        '$id' => ID::custom('schedules'),
-        'name' => 'schedules',
+        '$id' => ID::custom('rules'),
+        'name' => 'Rules',
         'attributes' => [
             [
-                '$id' => ID::custom('resourceType'),
+                '$id' => ID::custom('projectId'),
                 'type' => Database::VAR_STRING,
                 'format' => '',
-                'size' => 100,
+                'size' => Database::LENGTH_KEY,
                 'signed' => true,
-                'required' => false,
+                'required' => true,
                 'default' => null,
                 'array' => false,
                 'filters' => [],
             ],
             [
-                '$id' => ID::custom('resourceId'),
+                '$id' => ID::custom('projectInternalId'),
                 'type' => Database::VAR_STRING,
                 'format' => '',
                 'size' => Database::LENGTH_KEY,
                 'signed' => true,
-                'required' => false,
+                'required' => true,
                 'default' => null,
                 'array' => false,
                 'filters' => [],
             ],
             [
-                '$id' => ID::custom('resourceUpdatedAt'),
-                'type' => Database::VAR_DATETIME,
+                '$id' => ID::custom('domain'),
+                'type' => Database::VAR_STRING,
                 'format' => '',
-                'size' => 0,
-                'signed' => false,
-                'required' => false,
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => true,
                 'default' => null,
                 'array' => false,
-                'filters' => ['datetime'],
+                'filters' => [],
             ],
             [
-                '$id' => ID::custom('projectId'),
+                '$id' => ID::custom('resourceType'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 100,
```

Comment:
If "resourceType" is WhiteList(['api', 'function'])
Can be less than 100 chars?
If possible to be constant integers will be better.

Reply:
We have a few of these in different collections, let's fix it later, not very urgent.